.cirrus.yml

```diff
@@ -154,16 +154,18 @@ build_each_commit_task:
         cpu: 2
         memory: "4Gb"
         disk: 40
-        matrix:
-            image_name: "${FEDORA_CACHE_IMAGE_NAME}"
+        image_name: "${FEDORA_CACHE_IMAGE_NAME}"
 
     timeout_in: 30m
 
-    script:
-        - $SCRIPT_BASE/setup_environment.sh
-        - git fetch --depth $CIRRUS_CLONE_DEPTH origin $CIRRUS_BASE_BRANCH
-        - env GOPATH=/var/tmp/go/ make build-all-new-commits GIT_BASE_BRANCH=origin/$CIRRUS_BASE_BRANCH
+    setup_environment_script: $SCRIPT_BASE/setup_environment.sh
+    build_each_commit_script:
+        - 'git fetch --depth $CIRRUS_CLONE_DEPTH origin $CIRRUS_BASE_BRANCH'
+        - 'env GOPATH=/var/tmp/go/ make build-all-new-commits GIT_BASE_BRANCH=origin/$CIRRUS_BASE_BRANCH'
 
+    subcommand_manpages_script:
+        - 'make podman'
+        - './hack/podman-commands.sh'
```

Comment:
@mheon I thought this would be okay here, since we're also doing a build.  The 'make podman' should be a no-op, and is there mostly just for documentation.  I'm not opposed to moving it though.

Reply:
Yeah, this ought to be fine

Reply:
Just want to make sure the last build that ran has the standard build tags

Reply:
nevermind